content/events/onsna2019.md

```diff
@@ -0,0 +1,31 @@
++++
+title = "Network Service Mesh at Open Networking Summit NA 2019"
+link = "https://onsna19.sched.com"
+dates = "April 3-5, 2019"
+image = "/img/events/logo_ons_na.png"
+summary = "Network Service Mesh is joining Open Netowrking Summit in San Jose, California with project introduction and a panel session."
+eventlink = "https://events.linuxfoundation.org/events/open-networking-summit-north-america-2019/"
+date = "2019-04-02"
+publishDate ="2019-03-13"
++++
+
+Network Service Mesh will be present at the Open Networking Summit in San Jose, April 2-5 2019. Ed and Frederick are presenting the NSM introduction. A panel with several community members will discuss the application of Kubernetes and a Network Service Orchestrator with 5G and Edge in mind.
+
+
+Wednesday, April 3
+------------------
+* 3:10pm - 3:40pm - [Network Service Mesh: An Introduction] (https://sched.co/LKdR) - Ed Warnicke, Frederick Kautz
+
+
+Thursday, April 4
+-----------------
+* 11:50am - 12:20pm - [Panel Discussion: Towards Kubernetes as the Network Service Orchestrator] (https://sched.co/LKUa) - Heather Kirksey, Ramki Krishnan, Ed Warnicke, Jeffrey Saelens, Nikolay Nikolaev
+
+# Happy NSM Hour
+
+* TBD
+
+# Supporting Collatoral
+* Slides
+  * [Network Service Mesh Intro](https://docs.google.com/presentation/d/1Vzmhv5vc10NyAa08ny-CCbveo0_fWkDckbkCD_N0fPg/edit)
```

Comment:
We will probably update the slides... so not *quite* these slides :)

Reply:
Done.